components/Chips/src/MDCChipView.m

```diff
@@ -257,6 +262,18 @@ - (UIColor *)inkColor {
   return [self inkColorForState:UIControlStateNormal];
 }
 
+- (void)setEnableRippleBehavior:(BOOL)enableRippleBehavior {
+  _enableRippleBehavior = enableRippleBehavior;
+
+  if (enableRippleBehavior) {
+    [self.inkView removeFromSuperview];
```

Comment:
I believe you should be correcting the RippleView's frame to match the ChipView's bounds.  That should allow the autoresizing mask to work correctly.  If that's incorrect, please disable the autoresizing mask in the `init`.

Reply:
That is done in `layoutSubviews` and is not necessary here.

Reply:
Yes, I'm requesting that it be done once  when it changes (here) rather than explicitly overriding all the work done in `layoutSubviews` by the constraint system.

Reply:
+1 if we can rely on an autoresizing mask of width + height to accomplish what we need.

Reply:
Can you please point me to where an auto resizing mask for width and height is set for the MDCChipView instance? I am unable to find it.

Reply:
Done.